### PR TITLE
Create function for adding CSS classes

### PR DESCRIPTION
**Pull request by jason-zz**
With addCSSClass function, the CSS classes can be easily added in template by using $Link.addCSSClass('column class1 class2').

### CODE REVIEW

File: code/dataobjects/Link.php [hunk 1]
```diff
@@ -127,7 +143,8 @@ public function forTemplate(){
 		if($url = $this->getLinkURL()){
 			$title = $this->Title ? $this->Title : $url; // legacy
 			$target = $this->getTargetAttr();
-			return "<a href='$url' $target>$title</a>";	
+			$class = $this->cssClass ? Convert::raw2xml( $this->cssClass ) : '';
```
Comment (nyeholt) [inline code/dataobjects/Link.php]: Should this actually be raw2att ?

**Comment by jason-zz** [comment]
@nyeholt It looks like raw2attr call raw2xml. I have updated it to use raw2attr.

**Comment by sheadawson** [comment]
Hey thanks for this Jason! Sorry to be pedantic, but just two little things:

- The new function sets/overwrites, it doesn't add. Could you rename it setCSSClass? 
- If no css class is set, we shouldn't add an empty class attribute to the <a> tag

**Comment by jason-zz** [comment]
@sheadawson No problems. I just updated my commit. Fixed spacing for getTargetAttr function as well.

**Comment by sheadawson** [comment]
Awesome thanks!

**Pull request merged**